openedx/features/edly/utils.py

```diff
@@ -108,3 +108,23 @@ def get_enabled_organizations(request):
         return []
 
     return [studio_site_edx_organization]
+
+
+def create_user_link_with_edly_sub_organization(request, user):
+    """
+    Create edly user profile link with edly sub organization.
+
+    Arguments:
+        request (WSGI Request): Django request object
+        user (object): User object.
+
+    Returns:
+        object: EdlyUserProfile object.
+    """
+    edly_sub_org = request.site.edly_sub_org_for_lms
+    edly_user_profile, created = EdlyUserProfile.objects.get_or_create(user=user)
```

Comment:
Ideally we should access edly_user_profile from user directly like `user.edly_profile` (https://github.com/edly-io/edx-platform/pull/73/files) but we could have a use case where the users added previously will not have edly user profile.

https://docs.djangoproject.com/en/1.11/topics/db/examples/many_to_many/

```python
edly_user_profile, __ = EdlyUserProfile.objects.get_or_create(user=user)
edly_user_profile.edly_sub_organizations.add(edly_sub_org)
edly_user_profile.save()

return edly_user_profile

```